src/layout/grid/__test__/gridColumn.test.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * To run this test from the church-management/client folder, execute the following command:
+ * npx jest ./src/layout/grid/__test__/grid.test.js
```

Comment:
Should be: `npx jest ./src/layout/grid/__test__/gridColumn.test.js`

Reply:
I'll get it fixed.

Reply:
Fixed.

Reply:
Resolving.